src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -133,6 +141,16 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
     return this.hasChromiumWebRTC();
   }
 
+  supportsDependencyDescriptorRtpHeaderExtension(): boolean {
+    return this.hasChromiumWebRTC();
+  }
+
+  supportsScalableVideoCoding(): boolean {
+    return (
+      this.hasChromiumWebRTC() && this.engine() === 'Blink' && this.engineMajorVersion() >= 111
```

Comment:
That's a good point. I can ask Shi when he is back. I feel like it was just odd to look at `engineMajorVersion` but not `engine` lol. I'll put a comment.

Reply:
*also*, we could theoretically add `safari` to `hasChromiumWebRTC` since it keeps it up to date. That function is really confusion now that i think about it lol. Basically means: 'has libwebrtc version later then the one used in oldest supported chrome'.